azkaban-exec-server/src/test/java/azkaban/execapp/AzkabanExecutorServerTest.java

```diff
@@ -86,7 +87,9 @@ public void testInjection() throws Exception {
         new AzkabanCommonModule(props),
         new AzkabanExecServerModule()
     );
+    SERVICE_PROVIDER.unsetInjector();
```

Comment:
Looks like it isn't anymore! It was necessary when things were not guiced the right way, but it seems to be working now without the change. Thanks!!